machine_manager/models/machinery.py

```diff
@@ -0,0 +1,111 @@
+        " defined for the machine.")
+    serial_char = fields.Char('Product Serial #')
+    serial = fields.Many2one('stock.production.lot', string='Product Serial #',
+                             domain="[('product_id', '=', product)]")
+    model_type = fields.Many2one('machine.model', 'Type')
+    status = fields.Selection([('active', 'Active'), ('inactive', 'Inactive'),
+                               ('outofservice', 'Out of Service')],
+                              'Status', required=True, default='active')
+    ownership = fields.Selection([('own', 'Own'), ('lease', 'Lease'),
+                                  ('rental', 'Rental')],
+                                 'Ownership', default='own', required=True)
+    bcyl = fields.Float('Base Cycles', digits=(16, 3),
+                        help="Last recorded cycles")
+    bdate = fields.Date('Record Date',
+                        help="Date on which the cycles is recorded")
+    purch_date = fields.Date('Purchase Date',
+                             help="Machine's date of purchase")
+    purch_cost = fields.Float('Purchase Value', digits=(16, 2))
+    purch_partner = fields.Many2one('res.partner', 'Purchased From')
+    purch_inv = fields.Many2one('account.invoice', string='Purchase Invoice')
+    purch_cycles = fields.Integer('Cycles at Purchase')
+    actcycles = fields.Integer('Actual Cycles')
+    deprecperc = fields.Float('Depreciation in %', digits=(10, 2))
+    deprecperiod = fields.Selection([('monthly', 'Monthly'),
+                                     ('quarterly', 'Quarterly'),
+                                     ('halfyearly', 'Half Yearly'),
+                                     ('annual', 'Yearly')], 'Depr. period',
+                                    default='annual', required=True)
+    primarymeter = fields.Selection([('calendar', 'Calendar'),
+                                     ('cycles', 'Cycles'),
+                                     ('hourmeter', 'Hour Meter')],
+                                    'Primary Meter', default='cycles',
+                                    required=True)
+    warrexp = fields.Date('Date', help="Expiry date for warranty of product")
+    warrexpcy = fields.Integer('(or) cycles',
+                               help="Expiry cycles for warranty of product")
+    location = fields.Many2one('stock.location', 'Stock Location',
+                               help="This association is necessary if you want"
+                               " to make repair orders with the machine")
+    enrolldate = fields.Date('Enrollment date', required=True,
+                             default=lambda
+                             self: fields.Date.context_today(self))
+    ambit = fields.Selection([('local', 'Local'), ('national', 'National'),
+                              ('international', 'International')],
+                             'Ambit', default='local')
+    card = fields.Char('Card')
+    cardexp = fields.Date('Card Expiration')
```

Comment:
'Card' field is used if the machine has any official document and its expiration date, maybe is not something usual.I can remove it if you think that does not apply.